conans/client/command.py

```diff
@@ -540,7 +557,8 @@ def package(self, *args):
         parser.add_argument("package", nargs="?", default="",
                             help='Package ID to regenerate. e.g., '
                                  '9cf83afd07b678d38a9c1645f605875400847ff3'
-                                 ' If not specified, ALL binaries for this recipe are re-packaged')
+                                 ' This optional parameter is only used for the local conan '
```

Comment:
Maybe "If not specified, ALL binaries for this recipe are re-packaged" should be maintained?